bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -311,6 +311,15 @@ private static String extractName(final GuideGroup group) {
             return tmpRows;
         }
 
+        // Update the auto row of a table.
+        // This is called when the auto row should be changed from disabled to initial.
+        void enableAutoRow(final TargetEnvironment env) {
+            final GroupRow autoGroupRow = new GroupRow(true, true, 0, env.getGuideEnvironment().automaticGroup(),
+                    Collections.emptyList());
+            rows.set(1, autoGroupRow);
+            fireTableRowsUpdated(1, 1);
```

Comment:
I believe that it will always run on the Swing thread, yes, as this code is only ever invoked by a mouse listener and an action listener. Does that sound right?